### PR TITLE
[GE] Grid - Visual Update

### PR DESCRIPTION
**Pull request by msDestiny14**
image

https://github.com/BabylonJS/ThePirateCove/issues/103

### CODE REVIEW

File: guiEditor/src/components/commandButtonComponent.tsx [hunk 1]
```diff
@@ -15,14 +17,23 @@ export class CommandButtonComponent extends React.Component<ICommandButtonCompon
     }
 
     public render() {
+       
+        const divClassName = this.props.altStyle ? 
+        `command-button-alt${this.props.disabled ? "-disabled" : ""}${this.props.isActive ? "-active" : ""}` :
+        `command-button ${this.props.isActive ? "active" : ""}`
```
Comment (RaananW) [inline guiEditor/src/components/commandButtonComponent.tsx:23]: This seems to be repeating (at least 3 times). can you reuse a single call?

File: guiEditor/src/components/propertyTab/propertyGrids/gui/gridPropertyGridComponent.tsx [hunk 2]
```diff
@@ -28,27 +37,36 @@ export class GridPropertyGridComponent extends React.Component<IGridPropertyGrid
...
+                            <TextInputLineComponent lockObject={this.props.lockObject} key={`rText${i}`} label="" icon={gridColumnIconDark} iconLabel={`Row ${i}`} value={rd} numbersOnly={true}
+                                onChange={(newValue) => {
+                                    this._rowDefinitions[i] = newValue;
+                                    this._rowEditFlags[i] = true;
+                                    this._editedRow = true;
+                                    this._removingRow = false;
+                                    this._rowChild = false;
+                                    this.forceUpdate();
+                                }} />
+                            <TextLineComponent tooltip="" label={`[${i}]`} value="" color="grey"></TextLineComponent>
```
Comment (RaananW) [inline guiEditor/src/components/propertyTab/propertyGrids/gui/gridPropertyGridComponent.tsx:58]: Is that the expected label? the index inside []?

**Review by RaananW — APPROVED** [review]
Just a few nits.

File: guiEditor/src/components/propertyTab/propertyGrids/gui/gridPropertyGridComponent.tsx (hunk 2 above)
Comment (msDestiny14) [reply guiEditor/src/components/propertyTab/propertyGrids/gui/gridPropertyGridComponent.tsx:58]: Yep it's part of the design: 
image

**Pull request merged**